mozilla_django_oidc_db/setup_configuration/steps.py

```diff
@@ -0,0 +1,53 @@
+from django_setup_configuration.configuration import BaseConfigurationStep
+from django_setup_configuration.exceptions import ConfigurationRunFailed
+
+from mozilla_django_oidc_db.forms import OIDCSetupConfigForm
+from mozilla_django_oidc_db.models import OpenIDConnectConfig
+from mozilla_django_oidc_db.setup_configuration.models import (
+    AdminOIDCConfigurationModel,
+)
+from mozilla_django_oidc_db.utils import create_missing_groups
+
+
+class AdminOIDCConfigurationStep(BaseConfigurationStep[AdminOIDCConfigurationModel]):
+    """
+    Configure admin login via OpenID Connect
+    """
+
+    verbose_name = "Configuration for admin login via OpenID Connect"
+    config_model = AdminOIDCConfigurationModel
+    namespace = "oidc_db_config_admin_auth"
+    enable_setting = "oidc_db_config_enable"
+
+    def execute(self, model: AdminOIDCConfigurationModel) -> None:
+
+        config = OpenIDConnectConfig.get_solo()
+
+        base_model_data = model.model_dump()
+        endpoint_config_data = base_model_data.pop("endpoint_config")
+
+        all_settings = {
```

Comment:
Default settings must be set since they are intended to be taken HTML form fields in a post request.
Otherwise you would have to set them in the form validate method.

But also, if there is an existent non-default value, it will use that instead of the default value. But if the value is already set, do you want to use it or the default value?

Reply:
I think the docs suggest it always uses the default value if not set which is not the case